HIP/hip-412.md

```diff
@@ -213,7 +233,149 @@ Note that mime types for directories are not uniformly defined. Some IPFS CIDs p
+		},
+		{
+			"locale": "jp",
+			"metadata_uri": "ipfs://yidooajiaefiakfldanm12554woakoakga4sfda"
+		}
+	]
+}
+```
+
+Here's a clarification for the most important fields for the `HIP412` format.
+
+#### files
+
+(Required) `files` array holds one or many files. Because the optional `image` property can be used as a preview image, the `files` array should at least hold one `file` object that represents the NFT. 
+
+#### attributes.trait_type
+
+(Required) Name of trait.
+
+### attributes.display_type
+
+(Optional) Indicates how the trait value should be displayed. Possible display types (but other types are allowed):
+
+- `text` (**default value representation**)
+- `percentage` (for integer or number values)
+- `boost` (for integer or number values)
+- `datetime` (for a number which represents the unix timestamp in seconds)
+- `date` (for a number which represents the unix timestamp in seconds)
+- `color` (for a hexadecimal or rgb string color sequence such as `#00ff44` or `rgb(0,255,0)`)
+
+
+#### attributes.value
+
+(Required) Value for trait. To give an example, imagine an NFT with the `trait_type: mouth`. Possible values are `bubblegum`, `smiling`, `braces`, or `trumpet`. 
+
+_Allowed types: string, integer, number, boolean_
+
+#### attributes.max_value
+
+(Optional) Adding a `max_value` sets a ceiling for a numerical trait's possible values. **NFT tooling should default this value to the maximum value seen for a collection.** Only use this property if you want to set a different value than the maximum value seen in the collection. Make sure the `max_value` is equal to or higher than the maximum value seen for your collection.
+
+_Allowed types: string, integer, number_
+
+#### localization.locale
+
+(Required) A two-letter language code identifying the file's language. No need to further define subregion locales such as `en-GB` to keep things simple.
+
+_Type: string (two-letter language code according to [ISO 639-1 standard](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes))_
+
+
+#### localization.is_default_locale
+
+(Optional) Indicates the primary language for this NFT. When multiple locales are listed, make sure to add this property to set the default locale for this metadata file. When `is_default_locale` is set, the `metadata_uri` property should not be defined for this localization object.
+
+_Type: boolean (false/true)_
+
+
+#### localization.metadata_uri
+
+(Required) CID or path to the localized NFT's metadata file. It's recommended to host your file on [IPFS](https://ipfs.io/) and use a service like [Pinata](https://pinata.cloud/) to easily pin your file. Your CID should look like this: `ipfs://<hash>`.
+
+Alternatively, you can use [Arweave](https://www.arweave.org/), receiving a similar CID that looks like this: `ar://<hash>`.
+
+When `is_default_locale` is set, don't define the `metadata_uri` property because it indicates the default language for this NFT's metadata.
```

Comment:
```suggestion
When `is_default_locale` is set, don't define the `uri` property because it indicates the default language for this NFT's metadata.
```